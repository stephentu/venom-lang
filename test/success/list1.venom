a = [1,2,3];
#print (a[0]);
print (a);
b = ['hello', "world"];
print (b);
class A
  attr name::string
  def self(name::string) = 
    self.name = name; 
  end
  def stringify() -> string = name; end
end
c = [A('Bob'), A('Bill'), A('Charlie')];
print (c);
