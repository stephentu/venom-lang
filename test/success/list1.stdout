[1, 2, 3]
[hello, world]
[Bob, Bill, Charlie]
